docs/diagnostics/PX1007.md

```diff
@@ -29,7 +32,22 @@ The diagnostic is not displayed for the following items:
  - Members of public entities (except nested public types and DAC property fields).
  - All members of a public entity if the entity has the `exclude` XML tag or is marked with the `PXHidden`, `PXInternalUseOnly`, or `Obsolete` attribute.
  - Class fields of DACs. 
- - System fields of DACs, which are the following: `CreatedByID`, `CreatedByScreenID`, `CreatedDateTime`, `LastModifiedByID`, `LastModifiedByScreenID`, `LastModifiedDateTime`, `TStamp`, and `NoteID`.
+ - Well-known DAC fields, which are the following:
+   - `Selected`
+   - `Attributes`
```

Comment:
Attributes and GroupMask are system fields with the special role in the platform